lib/config/folderconfiguration.go

```diff
@@ -73,28 +75,58 @@ func (f FolderConfiguration) Path() string {
 	return f.cachedPath
 }
 
+func (f FolderConfiguration) SyncthingPath() string {
+	return filepath.Join(f.Path(), ".syncthing")
+}
+
 func (f *FolderConfiguration) CreateMarker() error {
 	if !f.HasMarker() {
-		marker := filepath.Join(f.Path(), ".stfolder")
+		marker := f.SyncthingPath()
 		fd, err := os.Create(marker)
 		if err != nil {
 			return err
 		}
 		fd.Close()
-		osutil.HideFile(marker)
 	}
 
 	return nil
 }
 
 func (f *FolderConfiguration) HasMarker() bool {
-	_, err := os.Stat(filepath.Join(f.Path(), ".stfolder"))
+	_, err := os.Stat(f.SyncthingPath())
 	if err != nil {
 		return false
 	}
 	return true
 }
 
+func (f *FolderConfiguration) CreateDefaultIgnores() error {
+	if f.HasIgnores() {
+		l.Infoln("Not overriding existing ignores with default ignores")
+		return nil
+	}
+
+	fd, err := os.Create(filepath.Join(f.SyncthingPath(), "ignores.txt"))
+	if err != nil {
+		return err
+	}
+
+	ignores := "#include .stsharedignore\n" + ignore.DefaultIgnores + "\n"
```

Comment:
If I remember correctly a nonexisting #include will break the parsing of the includes right now. Also, should probably use a name more like sharedignores.txt if we're going to default to something? (And are includes relative to the folder root or the .syncthing dir?)